src/Orleans.Core.Abstractions/IDs/GrainId.cs

```diff
@@ -241,25 +235,20 @@ private string ToStringImpl(bool detailed)
                 case UniqueKey.Category.KeyExtGrain:
                     var typeString = TypeCode.ToString("X");
                     if (!detailed) typeString = typeString.Substring(Math.Max(0, typeString.Length - 8));
-                    fullString = String.Format("*grn/{0}/{1}", typeString, idString);
+                    fullString = $"*grn/{typeString}/{idString}";
                     break;
                 case UniqueKey.Category.Client:
-                    fullString = "*cli/" + idString;
+                    fullString = $"*cli/{idString}";
                     break;
                 case UniqueKey.Category.GeoClient:
-                    fullString = string.Format("*gcl/{0}/{1}", Key.KeyExt, idString);
+                    fullString = $"*gcl/{Key.KeyExt}/{idString}";
+                    break;
+                case UniqueKey.Category.SystemGrain:
+                    fullString = $"*/sgn/{Key.PrimaryKeyToGuid()}/{idString}";
```

Comment:
why do these and the have a `/` at the start?

Reply:
I probably need new eyes

Reply:
I mean, `*/blah` instead of just `*blah` like the others

Reply:
> I mean, */blah instead of just *blah like the others

Yes I saw it, my response was for the "why": I was certain that all cases had a `/` after the `*` :p

Reply:
got it ;)